kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -79,17 +77,14 @@ def get_kubeconfig_path():
 
 def get_label():
     test = os.environ.get("PYTEST_CURRENT_TEST")
+    test = test.split(" ")[0]  # if invoked from setup, will have suffix ' (setup)', e.g.
```

Comment:
```suggestion
    test = test.split(" ", 1)[0]  # if invoked from setup, will have suffix ' (setup)', e.g.
```

No need to do additional splits.